README/ReleaseNotes/v636/index.md

```diff
@@ -28,7 +28,8 @@ The following people have contributed to this new version:
  Manuel Tobias Schiller, University of Glasgow,\
  Surya Somayyajula, UMass Amherst,\
  Petr Stepanov, @petrstepanov,\
- Dongliang Zhang, University of Science and Technology of China
+ Dongliang Zhang, University of Science and Technology of China,\
+ Enrico Lusiani, INFN Padova
```

Comment:
Sorry I forgot to say that before: please insert your name in alphabetical order. Thanks!